src/appengine/handlers/cron/fuzzer_and_job_weights.py

```diff
@@ -20,19 +20,32 @@
 from base import utils
 from datastore import data_handler
 from datastore import data_types
+from datastore import ndb
 from datastore import ndb_utils
 from google_cloud_utils import big_query
 from handlers import base_handler
 from libs import handler
 from metrics import fuzzer_stats
 from metrics import logs
+from system import environment
 
 QuerySpecification = collections.namedtuple(
     'QuerySpecification', ['query_format', 'formatter', 'reason'])
 
 SpecificationMatch = collections.namedtuple('SpecificationMatch',
                                             ['new_weight', 'reason'])
 
+DEFAULT_MULTIPLIER = 10.0  # Used for blackbox and unrun jobs.
+DEFAULT_SANITIZER_WEIGHT = 0.1
+
+SANITIZER_WEIGHTS = {
+    'ASAN': 0.5,
```

Comment:
How about we use a factor of `SANITIZER_WEIGHTS` for all of these, e.g. `5 * SANITIZER_WEIGHTS`, `1 * SANITIZER_WEIGHTS`, etc?